salt/example.top

```diff
@@ -13,7 +13,7 @@ base:
         - elife.goaws
         - search.java
         - search
-        # - search.processes
+        - search.processes
```

Comment:
usually processes are not enabled by default because they have some PHP code running that may be out-of-date and that interacts with databases, for example affecting a test suite being run. Alternative to start them would be to run a single command, `sudo start search-processes`

Reply:
Means a `vagrant up` results in a provision failure as is.

Reply:
Can't reproduce by patching this branch with:
```
diff --git a/salt/example.top b/salt/example.top
index 9e19471..48a2edc 100644
--- a/salt/example.top
+++ b/salt/example.top
@@ -13,7 +13,7 @@ base:
         - elife.goaws
         - search.java
         - search
-        - search.processes
+        #- search.processes
         - api-dummy
         - search.elasticsearch
         - elife.proofreader-php
```

Reply:
Think my memory was playing tricks on me. It was `c13cf6387000402cd70b099006a01a6be379a083` causing the failure; not having the processes running stops the tests from working IIRC.

Reply:
Just tried it out, some tests do not run because they make assumptions that a `search--ci` queue exist. Fixing that separately.